libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -168,14 +172,9 @@ int AP_Logger_Write(lua_State *L) {
         return luaL_argerror(L, args, "unknown format");
     }
 
-    lua_Alloc allocf = lua_getallocf(L, nullptr);
-    char *buffer = (char*)allocf(nullptr, nullptr, 0, msg_len);
+    char *buffer = (char*)luaM_malloc(L, msg_len);
     if (buffer == nullptr) {
```

Comment:
This check is now unneeded, `luaM_malloc` will throw the appropriate errors.

Reply:
fixed